### PR TITLE
LPS-155427

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://issues.liferay.com/browse/LPS-155427

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Review by miltonmc — APPROVED** [review]
The frontend LGTM!

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java [hunk 1]
```diff
@@ -176,10 +162,43 @@ public ObjectAction updateObjectAction(
...
 			throw new ObjectActionNameException();
 		}
+
+		if (!_objectActionExecutorRegistry.hasObjectActionExecutor(
+				objectActionExecutorKey)) {
+
+			throw new ObjectActionExecutorKeyException(objectActionExecutorKey);
+		}
+
+		if (!Objects.equals(
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java]: Done!

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java [hunk 2]
```diff
@@ -176,10 +162,43 @@ public ObjectAction updateObjectAction(
...
+				objectActionTriggerKey,
+				ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE)) {
+
+			if (!_messageBus.hasDestination(objectActionTriggerKey)) {
+				throw new ObjectActionTriggerKeyException();
+			}
+
+			if (Validator.isNotNull(conditionExpression)) {
+				throw new ObjectActionConditionExpressionException(
+					StringBundler.concat(
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java]: Done!

**Review by rodrigopaulino — APPROVED** [review]
LGTM! 👏🏻

**Pull request closed without merging**